PopPUNK/visualise.py

```diff
@@ -419,14 +418,15 @@ def generate_visualisations(query_db,
                 mode = "lineages"
                 suffix = "_lineages.csv"
             cluster_file = os.path.join(model_prefix, os.path.basename(model_prefix) + suffix)
-        isolateClustering = readIsolateTypeFromCsv(cluster_file,
-                                                   mode = mode,
-                                                   return_dict = True)
+
+    isolateClustering = readIsolateTypeFromCsv(cluster_file,
+                                               mode = mode,
```

Comment:
Given that this was an issue before, perhaps somewhere above/clearly setting a default value for mode would be sensible

Reply:
Agree, but can we defer to a future time?

Reply:
Looking through the context to this more and I think it's fine now